runtime/protocol/http-client-engines/http-client-engine-default/common/src/aws/smithy/kotlin/runtime/http/engine/DefaultHttpEngine.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package aws.smithy.kotlin.runtime.http.engine
+
+/**
+ * Factory function to create a new HTTP client engine using the default for the current KMP target
+ */
+fun SdkHttpEngine(config: HttpClientEngineConfig = HttpClientEngineConfig.Default): HttpClientEngine =
+    newDefaultHttpEngine(config)
+
+fun SdkHttpEngine(block: HttpClientEngineConfig.Builder.() -> Unit): HttpClientEngine {
+    val builder = HttpClientEngineConfig.Builder().apply(block)
+    val config = HttpClientEngineConfig(builder)
+    return SdkHttpEngine(config)
+}
```

Comment:
**Comment**: The name `SdkHttpEngine` feels ambiguous and inaccurate to me. Technically the CRT engine is also an "SDK HTTP Engine". It'd be clearer to me if it were `DefaultHttpEngine` or something similar.